src/i18n/en-US.properties

```diff
@@ -54,6 +54,8 @@ find_next=Find next
 find_previous=Find previous
 # Button tooltip for closing the find bar
 find_close=Close
+# Search input Aria-label attribute
+search_input=Search input
```

Comment:
Can we use `find_label` to stay consistent with the other `find_` keys? That will keep them all together in the output JSON file, as well.

Reply:
@jstoffan I'll change this as well